buddyboss/buddyboss-platform
# Enable copy paste HTML for medium editor

### All Submissions:






### Changes proposed in this Pull Request:
Disable `cleanPastedHTML` flag in the mediumEditor library. It fixes the copy paste content issue.

Fixes #1688 

### How to test the changes in this Pull Request:

1. Copy the content from editor like Gutenberg
2. Paste that content in the activity editor.
3. See result

### Proof Screenshots or Video
https://www.awesomescreenshot.com/video/5041992?key=fb401b2adfdb871c59b6695801a7fa49

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
